capstone-frontend/src/components/SearchUserModal.jsx

```diff
@@ -135,7 +135,7 @@ export class SearchUserModal extends Component {
       remove_collaboratorsIDs: user.id,
     }
 
-    // Remove Collaborator to Booklist in Firebase
+    // Remove BookList in Firebase
```

Comment:
Why was this changed? I feel like the original comment was more relevant

Reply:
This was probably a merge error I overlooked.